src/app/reservation/create-reservation/floor-and-wing-step/floor-and-wing-step.component.html

```diff
@@ -0,0 +1,29 @@
+<form class="pt-0 md:pt-24">
+  <ng-template matStepLabel>Kies verdieping en vleugel</ng-template>
+  <h4 class="text-2xl font-extrabold">Verdieping:</h4>
+  <mat-form-field>
+    <mat-label>Kies verdieping</mat-label>
+    <mat-select>
+      @for (floor of floors; track floor){
+        <mat-option value="{{floor.number}}"
+                    (onSelectionChange)="
+                    onFloorSelectionChange(floor);"
+        >{{floor.number}}</mat-option>
+      }
+    </mat-select>
+  </mat-form-field>
+  <h4 class="text-2xl font-extrabold">Vleugel:</h4>
+  <mat-form-field>
```

Comment:
Is er een reden dat dit niet in het midden staat? De rest van de stappen hebben dat wel namelijk